src/InternPool.zig

```diff
@@ -2985,14 +3048,25 @@ pub const Tag = enum(u8) {
     /// Trailing:
     /// 0. type: Index for each fields_len
     /// 1. name: NullTerminatedString for each fields_len
-    /// 2. init: Index for each fields_len // if tag is type_struct_packed_inits
+    ///
+    /// if tag is type_struct_packed_inits:
+    /// 2. body_indices: Zir.Inst.Index // for each field in declared order, a pair of start index and length.
```

Comment:
This information is redundant, and should not be duplicated in the InternPool. Instead, it should be re-fetched from ZIR extra data when resolving field inits. Duplicating this information in the pool is particularly undesirable as it may change across incremental updates.

Merge blocker.

Reply:
Ah, I hadn't considered incremental compilation. I'll duplicate the logic from `semaStructFields` that works through the ZIR extra data instead.

Reply:
I've reworked this to re-fetch the init body ranges. I moved some common preamble logic into `semaStructFieldsLen`.